env/requirements-style.txt

```diff
@@ -4,3 +4,8 @@ flake8
 pylint>=2.4
 pathspec
 burocrata
+types-requests
+types-tqdm
+types-paramiko
+mypy
+pytest
```

Comment:
Is there a reason why we need `pytests` for checking styles?

Reply:
Mypy is run over the test case files as well, so this needs pytest installed to recognize the import.